### PR TITLE
Update release references

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Review by marquiz — COMMENTED** [review]
You want to update `operand_version` in `config/manager/manager.yaml` as well.

Also the references in  `config/manager/manager.yaml` and `config/manifests/bases/nfd-operator.clusterserviceversion.yaml`(?)

We really should probably have a script under `hack/` to do these, I think. Smth similar to `prepare-release.sh` in nfd or anything

**Comment by marquiz** [comment]
What about `config/manifests/bases/nfd-operator.clusterserviceversion.yaml`, should(n't) it be updated, too?

**Comment by ArangoGutierrez** [comment]
> What about `config/manifests/bases/nfd-operator.clusterserviceversion.yaml`, should(n't) it be updated, too?

It is

File: config/manifests/bases/nfd-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -463,8 +463,8 @@ spec:
                 - name: OPERATOR_NAME
                   value: cluster-nfd-operator
                 - name: NODE_FEATURE_DISCOVERY_IMAGE
-                  value: gcr.io/k8s-staging-nfd/node-feature-discovery:master-minimal
-                image: gcr.io/k8s-staging-nfd/node-feature-discovery-operator:master-minimal
+                  value: k8s.gcr.io/nfd/node-feature-discovery:v0.11.0-minimal
+                image: k8s.gcr.io/nfd/node-feature-discovery-operator:v0.5.0-minimal
```
Comment (ArangoGutierrez) [inline config/manifests/bases/nfd-operator.clusterserviceversion.yaml:467]: @marquiz ?

Comment (ArangoGutierrez) [reply config/manifests/bases/nfd-operator.clusterserviceversion.yaml:467]: @marquiz ping?

**Review by marquiz — APPROVED** [review]
👌
/lgtm

**Comment by ArangoGutierrez** [comment]
Hello mr Prow....
/woof

**Pull request merged**